src/SiteMaster/Core/Auditor/Site/Reviews/AllForSite.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace SiteMaster\Core\Auditor\Site\Reviews;
+
+use SiteMaster\Core\InvalidArgumentException;
+
+class AllForSite extends All
+{
+    public function __construct(array $options = array())
+    {
+        if (!isset($options['sites_id'])) {
+            throw new InvalidArgumentException('the sites_id must be set', 500);
+        }
+
+        parent::__construct($options);
+    }
+
+    public function getSQL()
+    {
+        //Build the list
+        $sql = "SELECT id as id
+                    FROM site_reviews
+                    WHERE site_reviews.sites_id = " . (int)$this->options['sites_id'] . "
+                    ORDER BY site_reviews.date_scheduled DESC";
+
+        return $sql;
+    }
+
+    public function getWhere()
+    {
+        return "WHERE scans_id = " . (int)$this->options['scans_id'];
```

Comment:
The `getSQL` override above essentially makes this method obsolete. It should be removed or the `getSQL` method should be refactored to used the `getWhere` and `getOrderBy` methods.